src/js/rise-cache.js

```diff
@@ -44,10 +45,21 @@ RiseVision.Common.RiseCache = (function () {
             // call ping again so correct ping URL is used for Rise Cache V2
             return self.ping(callback);
           } else {
-            console.debug("Rise Cache is not running");
-            _isCacheRunning = false;
 
-            callback(false, null);
+            if ( _isHttp ) {
```

Comment:
@stulees good question, it will be something else but 404. It returns ERR_EMPTY_RESPONSE. I tested myself. cheers